pallets/murmur/Cargo.toml

```
@@ -7,7 +7,7 @@ homepage = "https://idealabs.network"
 edition = "2021"
 license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
 publish = false
-repository.workspace = true
+repository = "https://github.com/ideal-lab5/etf/"
```

```suggestion
repository = "https://github.com/ideal-lab5/pallets/"
```
Thought we should rename the repo...